prov/efa/src/efa_domain.c

```diff
@@ -206,9 +207,10 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 {
 	struct efa_domain *domain;
 	struct efa_fabric *fabric;
+	struct rxr_domain *rxr_domain;
 	const struct fi_info *fi;
 	size_t qp_table_size;
-	int ret;
+	int ret, enable_cache = 0;
```

Comment:
Libfabric appears to require the bool type; we should use it for `enable_cache`.

Reply:
I have to make a change in a future PR here so I can resolve this.